src/schema/v2/OrderedSet/orderedSetsConnection.ts

```diff
@@ -0,0 +1,61 @@
+import { GraphQLFieldConfig, GraphQLString } from "graphql"
+import { ResolverContext } from "types/graphql"
+import { paginationResolver } from "schema/v2/fields/pagination"
+import { pageable } from "relay-cursor-paging"
+import { convertConnectionArgsToGravityArgs } from "lib/helpers"
+import { OrderedSetConnection } from "./OrderedSet"
+
+export const OrderedSetsConnection: GraphQLFieldConfig<
+  void,
+  ResolverContext
+> = {
+  type: OrderedSetConnection.connectionType,
+  description: "A list of Ordered Sets",
+  args: pageable({
+    term: {
+      type: GraphQLString,
+      description: "If present, will search by term",
+    },
+  }),
+  resolve: async (_root, args, { matchSetsLoader, setsLoader }) => {
+    if (!matchSetsLoader)
```

Comment:
I think this is ok, b/c logged out users aren't expected to query `orderedSetsConnection`, but technically the Gravity endpoint does not enforce any  authorization on the endpoint that backs `setsLoader`, so we _might_ want to allow? Basically, I think I'm suggesting changing this to `if (!!args.term && !matchSetsLoader)`

Reply:
Good call. Will update.